ome-demoserver.yml

```diff
@@ -298,13 +289,22 @@
     - name: Download the Figure_To_Pdf.py script
       become: yes
       get_url:
-        url: https://raw.githubusercontent.com/ome/omero-figure/{{ omero_figure_tag | default("master") }}/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py
+        url: https://raw.githubusercontent.com/ome/omero-figure/v{{ omero_figure_release }}/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py
```

Comment:
`v` could be added to the string as part of the default to still permit `master`.

Reply:
we will need a new parameter in that case since we use the version to install from pypi 
and the version to retrieve the tag from GitHub